pkg/apis/veleroplugin/v1/upload.go

```diff
@@ -28,18 +28,22 @@ type UploadSpec struct {
 	// BackupTimestamp records the time the backup was called.
 	// The server's time is used for SnapshotTimestamp
 	BackupTimestamp *meta_v1.Time `json:"backupTimestamp,omitempty"`
+
+	// UploadCancel indicates request to cancel ongoing upload.
+	UploadCancel bool `json:"uploadCancel,omitempty"`
 }
 
 // UploadPhase represents the lifecycle phase of a Upload.
-// +kubebuilder:validation:Enum=New;InProgress;Completed;UploadError;CleanupFailed
+// +kubebuilder:validation:Enum=New;InProgress;Completed;UploadError;CleanupFailed;Canceled;
 type UploadPhase string
 
 const (
 	UploadPhaseNew           UploadPhase = "New"
 	UploadPhaseInProgress    UploadPhase = "InProgress"
 	UploadPhaseCompleted     UploadPhase = "Completed"
-	UploadPhaseUploadError  UploadPhase = "UploadError"
+	UploadPhaseUploadError   UploadPhase = "UploadError"
 	UploadPhaseCleanupFailed UploadPhase = "CleanupFailed"
+	UploadPhaseCanceled 	 UploadPhase = "Canceled"
```

Comment:
I think we should have a Canceling phase as well.  Once you set the flag, you don't know if the controller is reacting to it or not.